starwars/src/App.js

```diff
@@ -12,6 +36,11 @@ const App = () => {
   return (
     <div className="App">
       <h1 className="Header">React Wars</h1>
+      <div className='cards'>
+      {results.map(person => {
+        return <Card key={person} person={person}/>;
```

Comment:
I love that you're passing down the whole person object instead of passing 5 different props.